scripts/zones/Wajaom_Woodlands/Zone.lua

```diff
@@ -16,6 +16,8 @@ end
 zone_object.onInitialize = function(zone)
     xi.helm.initZone(zone, xi.helm.type.HARVESTING)
     xi.chocobo.initZone(zone)
+    UpdateNMSpawnPoint(ID.mob.HYDRA)
+    GetMobByID(ID.mob.HYDRA):setRespawnTime(math.random(48, 72) * 3600)
```

Comment:
would you even need this line to spawn it on startup + time unless after a mait to reset times, shouldnt the persist data handle the spawn from a restart?

Reply:
How is the initial spawn supposed to happen if respawn time is set to 0 in groups and the zone init doesn't manually set the time?

Reply:
i committed removal of the two lines referenced by @KnowOne134 . Admittedly im traveling starting tomorrow and won't have much time to look at this until next week, if they need to be re-added let me know, just trying to get my affairs in order before leaving ^^;

Reply:
The persist data will overwrite this respawn time if it exists, but it still needs this for the first time initial spawn (and if you're not using the module).

Reply:
yea sorry mine was a question because im unfamiliar with lsb persist code and thinking will this overwrite the persist on everty restart